engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/NKGlobalParameters.scala

```diff
@@ -0,0 +1,34 @@
+package pl.touk.nussknacker.engine.flink.api
+
+import java.util
+import scala.collection.JavaConverters._
+
+import org.apache.flink.api.common.ExecutionConfig
+import org.apache.flink.api.common.ExecutionConfig.GlobalJobParameters
+
+//we can use this class to pass config through RuntimeContext to places where it would be difficult to use otherwise
+//Also, those configuration properties will be exposed via Flienk REST API/webconsole
+//TODO: 
+case class NKGlobalParameters(buildInfo: String,
+                              versionId: Long,
+                              modelVersion: Option[Int],
+                              user: String,
+                              useLegacyMetrics: Option[Boolean]) extends GlobalJobParameters {
+
+  override def toMap: util.Map[String, String] = {
+    getClass.getDeclaredFields.map(_.getName).zip(productIterator.toList).toMap.mapValues {
+      case None => "none"
+      case Some(x) => x.toString
+      case null => "null"
+      case x => x.toString
+    }.asJava
+  }
+}
+
+object NKGlobalParameters {
+
+  def readFromContext(ec: ExecutionConfig): Option[NKGlobalParameters] = Option(ec.getGlobalJobParameters).collect {
```

Comment:
Shouldn't we read some of params like `useLegacyMetrics` directly from type safe's config and pass to global parameters only those that are determined on ui side (like `version`)? I wonder if this `getGlobalJobParameters` won't grow too much when we pass a lot of options (especially in nested structures, lists etc.)

Reply:
hmm... true that... I'll prepare sth

Reply:
addressed in https://github.com/TouK/nussknacker/pull/884